### ISSUE TITLE
[gulp] changes are not visible until I do "Flush all caches"

### ISSUE THREAD

**Issue by partikus** [droptica/droopler#672]
Dear Droopler Team!

First of all, thank you for sharing Droopler with PHP community. 

I've installed a droopler including demo data. Installation process was done fine. Then I've launched a `gulp watch` task for subtheme and all sass compilations are being done really well but ...
unfortunately, the recompiled css files are not available in the browser until I force clearing cache.

`Flush all caches` takes ages so I'm wondering if it's possible to get the up-to-date version of assets.

Please let me know I'm doing wrong or which settings should be changed to achieve it.

Cheers!

**Comment by grzegorz-pietrzak-droptica** [comment]
Hi, please disable css/js aggregation in the Performance configuration (please note that you should do it also for AdvAgg module).

**Comment by partikus** [comment]
@grzegorz-pietrzak-droptica thank you 👍 that's what I did. 

1. I added the following settings to configuration:
`$config['system.performance']['css']['preprocess'] = FALSE;`
`$config['system.performance']['js']['preprocess'] = FALSE;`

2. But it did not help. Then I've found AdvAgg module under performance tab.
After applying the following configuration: 
`Enable advanced aggregation: Disabled` 
`AdvAgg Cache Settings: Development`
The problem has been solved.

I'm wondering if there is a way to turn this off in `settings.local.php` or so.

Anyway thank you for your support 👏 👍 🥇
